src/Tooltip.js

```diff
@@ -113,12 +116,20 @@ class Tooltip extends React.Component {
     return delay;
   }
 
+  getTarget() {
+    const { target } = this.props;
+    if (typeof target === 'object') {
+      return target;
+    }
+    return document.getElementById(target);
+  }
+
   getTetherConfig() {
     const attachments = getTetherAttachments(this.props.placement);
     return {
       ...defaultTetherConfig,
       ...attachments,
-      target: '#' + this.props.target,
+      target: this.getTarget(),
```

Comment:
I forgot that I used the string selector because I wasn't sure if the target element would be available at the time getTetherConfig is called. Tether also accepts a function as target. This might be safer as:

```
target: this.getTarget
```

`this._target = this.getTarget()` in `componentDidMount` should be fine since it's mounted.